package.json

```diff
@@ -23,6 +23,11 @@
   },
   "homepage": "https://github.com/ballercat/wasm-loader#readme",
   "dependencies": {
+    "loader-utils": "^1.1.0",
+    "wasm-dce": "^1.0.0"
+  },
+  "peerDependencies": {
+    "wasm-dce": "1.x"
```

Comment:
I don't think we need the `dependencies` entry any longer.

Reply:
It told me that `npm WARN wasm-loader@1.1.0 requires a peer of wasm-dce@1.x but none is installed. You must install peer dependencies yourself.`. We can add an installation step in the README.

Reply:
We do the same in babel-loader:

https://github.com/babel/babel-loader/blob/master/package.json#L17-L23